slsc-face-archive.md

```diff
@@ -0,0 +1,15 @@
+# Project SLSC Photo Archive Face Tagging
+
+## Client Profile
+
+Youth Exploring Science (YES) program is created to help underserved students in St. Louis area. Students participate in YES program to gain skills in Science, Technology, Engineering, Arts and Mathematics fields. High school students will gain real-world skills, career readiness training through YES program. Students in the YES program also promote the importance of STEAM-related careers.
+
+## Project Brief
+
+The SLSC Face Archive Project captures and preserves various facial expressions and emotions, assisting researchers, psychologists, and developers in disciplines such as mental health, AI, and emotion identification. It adheres to ethical norms, ensuring participant consent, privacy, and anonymity, and has a wide range of applications, including investigating emotional responses and refining AI algorithms.
```

Comment:
To briefly describe about the project discussion we had, we thought of having a dataset which consists of images of various people linked with their names. When this dataset is passed to an ML model, the model should be able to recognize the face of each student in the image with their corresponding names.
Later, we have collaborated with the professor Jasmine, the person who is closely working on the face tagging project. We found out that, in Youth Exploring Science program, there are thousands of photos of students. I came to know that most of the photos are on Dropbox, flicker and google photos. Currently the photos are organized by event and date, So, we should be able to find a way where we could be able to sort photos by student names and face which makes easy to collect information of each student without looking up the dates and events.
So, we have thought of the following steps,
1. Collecting the dataset.
2. Dividing the dataset to training and testing dataset.
3. Creating a model for face recognition.
4. Based on the pre defined trained dataset as input, the model predicts the student details with their face.

Reply:
incorporated comments into updates.

Reply:
> To briefly describe about the project discussion we had, we thought of having a dataset which consists of images of various people linked with their names. When this dataset is passed to an ML model, the model should be able to recognize the face of each student in the image with their corresponding names. Later, we have collaborated with the professor Jasmine, the person who is closely working on the face tagging project. We found out that, in Youth Exploring Science program, there are thousands of photos of students. I came to know that most of the photos are on Dropbox, flicker and google photos. Currently the photos are organized by event and date, So, we should be able to find a way where we could be able to sort photos by student names and face which makes easy to collect information of each student without looking up the dates and events. So, we have thought of the following steps,
> 
> 1. Collecting the dataset.
> 2. Dividing the dataset to training and testing dataset.
> 3. Creating a model for face recognition.
> 4. Based on the pre defined trained dataset as input, the model predicts the student details with their face.

I can use Python to scrap the data (face and save them with their names in CSV format)..i would just need the the URL of the youth exploring science program to dod so..  before we split them into test and training sets(say 80-20)... so we can have like gallery(combination of test and train) of images and find out from the gallery which matches in the test set. in order to deduce our model's accuracy...